templates/content/GeoBlazor.Template.WebApp/GeoBlazor.Template.WebApp/GeoBlazor.Template.WebApp.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
There seems to be two whole copies of `GeoBlazor.Template.WebApp`, one inside the other one.